.github/workflows/test-integration-17.yml

```diff
@@ -0,0 +1,37 @@
+name: Integration@node 17
+on:
+  pull_request:
+  push:
+    branches-ignore:
+      - 'gh-pages'
+
+jobs:
+  test-integration-17:
+    runs-on: ubuntu-latest
+
+    services:
+      redis:
+        image: redis
+        options: >-
+          --health-cmd "redis-cli ping"
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        ports:
+          - 6379:6379
```

Comment:
this seems mildly annoying to have to setup ourselves. did the circle image have all this baked in already?

Reply:
I think both Circle and GHA are just using the `redis` image from DockerHub.

We definitely do need to explicitly map the port on GitHub Actions - if you remove that it doesn't work. Not sure why we didn't have to do that on Circle.

With the health checks, I put them in because all the examples on https://docs.github.com/en/actions/using-containerized-services/creating-redis-service-containers have it but the tutorial doesn't really explain it beyond `# Set health checks to wait until redis has started`. I could try deleting it and see if it still works.